src/server/db_slice.h

```diff
@@ -598,6 +601,7 @@ class DbSlice {
   size_t soft_budget_limit_ = 0;
   size_t table_memory_ = 0;
   uint64_t entries_count_ = 0;
+  size_t load_in_progress_ = 0;
```

Comment:
nit: use size_t for sizes, unsigned for counters, also maybe change the name to `load_ref_cnt_` ?

Reply:
sure no issue again that! Out of curiosity why unsigned for counter ? (size_t is also unsigned :smile: )

Reply:
yeah, it's very subjective `size_t` and uint64_t are the same type but size_t tells me it's sizes, lengths etc.
counters - are usually uintxxx, that's how I try to declare types (proper readability aspect)

Reply:
Sounds good, I will keep it in mind/ apply the notation. Cheers